backend/src/api.rs

```diff
@@ -35,11 +35,9 @@ fn parse_songs(output: Vec<(i32, i32, String, String, i32, i32, String, i32, Str
                     title: row.2,
                     filetype: row.3,
                     media: row.4,
-                    genres: HashSet::new(),
```

Comment:
This should be added again next time - at this moment database doesn't support genres of songs